scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -409,6 +407,31 @@ public TaskResult getTaskResult(String jobId, String taskName) throws NotConnect
         return this.getTaskResult(JobIdImpl.makeJobId(jobId), taskName);
     }
 
+
+    @Override
+    public List<TaskResult> getTaskResultByTag(JobId jobId, String taskTag)
```

Comment:
Is it not possible to mark it with @ImmediateService. A method marked with `immediateService` can be executed in parallel to another. However, the business code should be thread-safe.